.travis.yml

```diff
@@ -151,6 +153,17 @@ matrix:
           tags: true
           repo: pantsbuild/pants
 
+    # Deploy Pex Unstable to s3
+    - name: "Deploy Pants PEX Unstable"
+      os: linux
+      language: python
+      stage: Deploy Pants Pex
+      env:
+        - PREPARE_DEPLOY=1
+#        - PANTS_PEX_RELEASE=stable
+      script:
+        - ./build-support/bin/release.sh -p && mv dist/*.pex dist/deploy/
```

Comment:
`release.sh -p` will build the pex at `dist/pants-<version>.pex`, so moving it to `dist/deploy/` would allow the deploy script to pick it up later.

This means it will put `pants-<version>.pex` at the top level of s3? Would it be alright or do you recommend somewhere else, like `s3:///unstable_pants_pex/pants-<version>.pex`, @jsirois ?

According to the travis doc, deploy script won't run for PR, so would appreciate some eyeballing here.

Reply:
We already have this general directory structure locally:
```
dist/
  deploy/
    wheels/
      3rdparty/
      pantsbuild.pants/
    bin/
      fsutil/
    pex/
```

I'd stick with it and mv into `dist/deploy/pex/`. Prefixing by type of binary meshes well with S3 support by "listing" (filtering) keys by prefix. Filtering by `pants.` seems a lot less robust going forward should we need to filter.

By definition, everything we deploy from pantsbuild.pants to S3 is unstable - we only deploy stable binaries there from pantsbuild.binaries. As such, including "unstable" in the prefix I think would only add confusion. It's true `fsutil` break the rule, being the only unstable `bin/` item, but better to leave that as the only thing not fitting the pattern I think.

Reply:
+1 to aligning with the existing directory structure that John mentioned.